### PR TITLE
KOGITO-4830: BPMN Read-only Mode E2E Coverage for standalone editors

### PR DESCRIPTION
**Pull request by domhanak**
Modify process-string.bpmn to include better strings for checks.
Add test to verify that BPMN editor correctlly handles read-only mode
for its properties.

@jstastny-cz peer review ready :)

### CODE REVIEW

**Comment by domhanak** [comment]
@jstastny-cz  one thing I am not sure about is handling reported issues that re not fixed. 
Should I add `// TODO: KOGITO-XXXX needs to be fixed ? ` or something similar? There are areas that are not disabled as expected.

**Comment by domhanak** [comment]
Tests will fail untill we have new version of editors published.

**Comment by tiagobento** [comment]
@domhanak Do you need me to generate a new snapshot version of the Editors from `master`?

File: packages/kie-editors-standalone/it-tests/cypress/integration/bpmn-read-only.spec.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
```
Comment (jstastny-cz) [inline packages/kie-editors-standalone/it-tests/cypress/integration/bpmn-read-only.spec.ts]: year 2021

**Review by jstastny-cz — COMMENTED** [review]
@domhanak I don't have a clear answer, I think we could do as follows:
1. the enabled test verifying the currently correctly working part of functionality.
2. a separate reproducing test case that is disabled for each of the open issues (plus comment)

**Comment by domhanak** [comment]
As discussed off-PR, I will work or reworking the structure of tests to allow skipping of part affected by issue.

**Comment by domhanak** [comment]
Does not seem like the tests are using the latest snapshot in it-tests. Even thought it is present in registry.

**Comment by domhanak** [comment]
@ederign @paulovmr Any suggestions what to do with this PR? It does not seem to use the latest snapshot as it should. 
It is available here:
https://www.npmjs.com/package/@kogito-tooling/bpmn-editor-unpacked

**Comment by domhanak** [comment]
Cool, so after the upgrade to 7.52.Final of editors tests are passing, will wait for the chrome ext to get fixed and I ll rebase once more.

**Comment by domhanak** [comment]
rebased

**Comment by domhanak** [comment]
failures in the build are unrelated, the issues is repo-wide and manifesting on all PR's

**Comment by domhanak** [comment]
@tomasdavidorg @jstastny-cz let me know if there is something to add

**Review by tomasdavidorg — APPROVED** [review]

**Comment by domhanak** [comment]
Checking the mac test failure, rerunning jobs. Please hold on until we get :green_circle:

**Review by jstastny-cz — APPROVED** [review]
LGTM, thanks.

**Comment by domhanak** [comment]
I see similar test failure here: https://github.com/kiegroup/kogito-tooling/pull/477/checks?check_run_id=2324626168

**Review by paulovmr — APPROVED** [review]

**Pull request merged**